File: src/main/java/eladkay/quaritum/common/item/misc/ItemRiftmakerPart.java
package eladkay.quaritum.common.item.misc;

import eladkay.quaritum.common.item.base.ItemMod;
import eladkay.quaritum.common.lib.LibNames;

public class ItemRiftmakerPart extends ItemMod {
    private static final String[] TYPES = new String[]{"riftFang", "riftFur", "riftClaw"};

    public ItemRiftmakerPart() {
        super(LibNames.RIFTMAKER_PART, TYPES);
        setMaxStackSize(1);
    }

}

File: src/main/java/eladkay/quaritum/common/item/misc/ItemRiftmakerPart.kt
package eladkay.quaritum.common.item.misc

import com.teamwizardry.librarianlib.common.base.item.ItemMod
import eladkay.quaritum.common.lib.LibNames

class ItemRiftmakerPart : ItemMod(LibNames.RIFTMAKER_PART, *ItemRiftmakerPart.TYPES) {

    init {
        setMaxStackSize(1)
    }

    companion object {
        private val TYPES = arrayOf("riftFang", "riftFur", "riftClaw")
    }

}
